source/common/http/conn_manager_impl.h

```diff
@@ -548,6 +548,7 @@ class ConnectionManagerImpl : Logger::Loggable<Logger::Id::http>,
       bool saw_connection_close_ : 1;
       bool successful_upgrade_ : 1;
       bool created_filter_chain_ : 1;
+      bool overloaded_do_not_create_filter_chain_ : 1;
```

Comment:
could you add a comment here?

Reply:
My understanding is that previous `created` actually have 2 cases, overload or normally created.
With the new upgrade failure, the former case envoy should reject but the latter case is legal to continue as upgrade_ignored

Reply:
created_filter_chain_ = true is set in createFilterChain as well, and the upgrade could fail, so there were at least 4 cases previously, if createFilterChain is called more than once.

Reply:
RE: the comment.  I would suggest that this is breaks the existing style.  If anything the name of this variable is more descriptive than any of the other similar (and uncommented) variables.